jvm-libs/linea/web3j-extensions/src/test/kotlin/net/consensys/linea/web3j/DomainObjectMappersTest.kt

```diff
@@ -1,5 +1,6 @@
 package net.consensys.linea.web3j
 
+/*
```

Comment:
Maybe we can just delete it right away?

Reply:
Yes, I will! I just didn't do to it yet until I have the test cases I the other test class.